lib/ui/select.tsx

```diff
@@ -108,19 +108,16 @@ const SelectItem = React.forwardRef<
   <SelectPrimitive.Item
     ref={ref}
     className={cn(
-      "relative flex w-full cursor-default select-none items-center rounded p-2 outline-none transition-colors focus:bg-f1-background-secondary focus:text-f1-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+      "relative grid w-full cursor-default select-none grid-cols-[1fr_20px] gap-x-1.5 rounded p-2 outline-none transition-colors focus:bg-f1-background-secondary focus:text-f1-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
       "data-[state=checked]:bg-f1-background-selected-bold/5 hover:data-[state=checked]:bg-f1-background-selected-bold/10",
       className
     )}
     {...props}
   >
-    <div className="absolute right-2 top-2 flex items-center justify-center">
-      <SelectPrimitive.ItemIndicator>
-        <CheckCircle className="h-5 w-5 text-f1-icon-selected" />
-      </SelectPrimitive.ItemIndicator>
-    </div>
-
     <SelectPrimitive.ItemText>{children}</SelectPrimitive.ItemText>
+    <SelectPrimitive.ItemIndicator className="flex">
+      <CheckCircle className="size-5 text-f1-icon-selected" />
+    </SelectPrimitive.ItemIndicator>
```

Comment:
I think using a `grid` here is a bit too much, we could use a simpler option like flex

Reply:
To achieve the same layout with flex, you'd need to add classes to the child elements, right? I used grid to avoid that, but if it's overkill I can change it, no probs! 😀

Reply:
I think both are fine and performance-wise it's not that one is more heavy than the other (trust me, I've asked ChatGPT, I'm an engineer)